lib/src/player_with_controls.dart

```diff
@@ -20,6 +20,16 @@ class PlayerWithControls extends StatelessWidget {
       return width > height ? width / height : height / width;
     }
 
+    Widget buildOverlay(BuildContext context) {
+      if(chewieController.overlayBuilder != null) {
+        return chewieController.overlayBuilder!(context);
+      }
+      if (chewieController.overlay != null) {
+        return chewieController.overlay!;
+      }
+      return const SizedBox.shrink();
```

Comment:
This should be chained as an if-else statement, with it defaulting to `SizedBox.shrink()`.

For bonus credit, instead of using a function to compute the correct Widget to return, why not move it into a standalone widget that takes in the current controller and computes it when `build` is called on it.